tests/results/96d93ac91d1dd3bb94db752f5b365efbd83411a0bddd09dfec1b848983b097bb/result.json

```diff
@@ -63,14 +81,22 @@
   "files": {
     "extracted": [
       {
-        "name": "Element[14]<embed>",
+        "name": "d87b0e79842cee502bbb1eb2161596fb7ca21db80dc58532dbd0a7e02828abb7",
```

Comment:
Have you found this to be suspicious on other samples? This is a gif and discussing about it, seems like a valid behaviour from legitimate websites.

Reply:
At a minimum, it should be de-based64 so that it doesn't make a branch in the tree just to be base64'd.